tox.ini

```diff
@@ -28,14 +28,18 @@ extras =
     lxc
     openstack
     vagrant
+commands_pre =
+    find {toxinidir} -type f -not -path '{toxinidir}/.tox/*' -not -path '*/__pycache__/*' -name '*.py[c|o]' -delete
```

Comment:
Why not `__pycache__`? I normally also delete such folders to remove as many side effects as possible.
Any objections?

Reply:
My impression is that __pycache__ folders are not affected by the same conflicts as the other files as they allow multiple interpreters to coexists without conflicts.

I copied this sequence from another project but I have nothing against removing this folders too if anyone knows that they can really cause conflicts.

Reply:
True. That's what I thought. Just wanted to be on the safe side :)